### PR TITLE
[docs] Add 'Learn More' section to 'Feature Preview' page

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

File: src/docs/feature-preview.md [hunk 1]
```diff
@@ -16,11 +16,17 @@ This feature also enables Docker in your workspace. Run `sudo docker-up` to star
...
 
 You can choose between 2 options:
 
 - Theia
 - Code
 
-> There's also a private beta for allowing a custom IDE image for the default IDE. If you are interested, feel free to reach out through the [contact form](https://www.gitpod.io/contact/).
+> There is also a private beta for allowing a custom IDE image for the default IDE. If you are interested, feel free to reach out through the [contact form](https://www.gitpod.io/contact/).
+
+## Learn More
```
Comment (gtsiolis) [inline src/docs/feature-preview.md:28]: **suggestion**: Using `h2` here makes this section look like a third feature in this list. Make we could move this under the first feature (Root Access) and use a smaller heading like `h4` as these are only references for more context. WDYT?

| BEFORE | AFTER |
|-|-|
| image | image |

**Review by gtsiolis — COMMENTED** [review]
Hey @jankeromnes! Added one comment related to content structure. Let me know what you think! 🏀

File: src/docs/feature-preview.md (hunk 1 above)
Comment (jankeromnes) [reply src/docs/feature-preview.md:28]: Makes sense, thanks!

**Comment by gtsiolis** [comment]
Alright, let's merge this! 🎉 

Thanks for adding this @jankeromnes!

**Pull request merged**